client/components/OrderPicker/OrderPicker.tsx

```diff
@@ -5,53 +5,47 @@ import Icon from 'components/Icon/Icon';
 
 require('./orderPicker.scss');
 
-type OwnProps = {
-	selectedItems: any[];
-	allItems: any[];
-	onChange: (...args: any[]) => any;
-	uniqueId: (...args: any[]) => any;
-	selectedTitle?: string;
-	availableTitle?: string;
-	selectedTitleTooltip?: string;
-	availableTitleTooltip?: string;
+type MinimalItem = {
+	id: string;
+	title: string;
+};
+
+type Props<Item> = {
+	selectedItems: Item[];
+	allItems: Item[];
+	onChange: (selectedItems: Item[]) => any;
+	uniqueId: string;
+	selectedTitle: React.ReactNode;
+	availableTitle: React.ReactNode;
+	selectedTitleTooltip: null | string;
+	availableTitleTooltip: null | string;
```

Comment:
I like the descriptive generic type argument names. I need to get better at that.